### ISSUE TITLE
[Feature Request] Device/Screen Sizes would be better than devices

### ISSUE THREAD

**Issue by lrh11099** [benvinegar/counterscale#119]
I think device/screen sizes show more actionable data than device names. You can even group them into mobile, tablets and computers 

Ref: https://github.com/benvinegar/counterscale/discussions/104

**Comment by tycobrah** [comment]
I support. This is like the last important feature that I personally would love to see in this project.

**Comment by tycobrah** [comment]
@benvinegar was this implemented in the latest RC? I am seeing the data being grouped like this proposal even though clicking on each returns empty results.

**Comment by benvinegar** [comment]
@tycobrah Yes, and thank you for reporting that. I completely forgot to test filters. 🙈

**Comment by singingblacks** [comment]
This issue should be closed? @benvinegar  @stordahl

**Comment by stordahl** [comment]
@singingblacks thanks for flagging!
